site/content/en/docs/user-manual/command-reference/searcher.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Search datasets'
+linkTitle: 'search'
+description: ''
+---
+
+This command search similar data results for query on dataset. The result includes top-k similar data among target dataset and the visualization of result is saved as png file. This tool is supposed to help users to figure out dataset property.
```

Comment:
There are mix usages of the two words: `top-k` and `topK`. Is it better to choose one of them to make it clean?

Reply:
I use `topk` to match the term in CLI. If you think Please let me know if you think the other one is better.

Reply:
I think `top-k` is more natural when used in the natural language sentence.

Reply:
I updated to use `top-k` in documentation, and still use `topk` in CLI.